tests/e2e/unified_test.go

```diff
@@ -173,42 +222,61 @@ func TestAllInSeries(t *testing.T) {
 						return false
 					})
 
-					jsonMutators := []test.JSONMutator{}
-
-					jsonMutators = append(jsonMutators, func(obj map[string]any) {
-						_, found, _ := unstructured.NestedString(obj, "uniqueId")
-						if found {
-							unstructured.SetNestedField(obj, "111111111111111111111", "uniqueId")
-						}
-					})
-					jsonMutators = append(jsonMutators, func(obj map[string]any) {
-						_, found, _ := unstructured.NestedString(obj, "oauth2ClientId")
-						if found {
-							unstructured.SetNestedField(obj, "888888888888888888888", "oauth2ClientId")
-						}
-					})
-					jsonMutators = append(jsonMutators, func(obj map[string]any) {
-						_, found, _ := unstructured.NestedString(obj, "etag")
-						if found {
-							unstructured.SetNestedField(obj, "abcdef0123A=", "etag")
+					// Remove operation polling requests (ones where the operation is not ready)
+					events.RemoveRequests(func(e *test.LogEntry) bool {
+						if !strings.Contains(e.Request.URL, "/operations/${operationID}") {
+							return false
 						}
-					})
-					jsonMutators = append(jsonMutators, func(obj map[string]any) {
-						_, found, _ := unstructured.NestedString(obj, "serviceAccount", "etag")
-						if found {
-							unstructured.SetNestedField(obj, "abcdef0123A=", "serviceAccount", "etag")
+						responseBody := e.Response.ParseBody()
+						if responseBody == nil {
+							return false
 						}
+						done, _, _ := unstructured.NestedBool(responseBody, "done")
+						return !done // remove if not done
 					})
+
+					jsonMutators := []test.JSONMutator{}
+					addReplacement := func(path string, newValue string) {
+						tokens := strings.Split(path, ".")
+						jsonMutators = append(jsonMutators, func(obj map[string]any) {
+							_, found, _ := unstructured.NestedString(obj, tokens...)
+							if found {
+								unstructured.SetNestedField(obj, newValue, tokens...)
+							}
+						})
+					}
+
+					addReplacement("uniqueId", "111111111111111111111")
```

Comment:
This method is already super long. Can we turn these "normalizations" to a helper function, like `[IgnoreComments](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/tests/samples/create/harness.go#L529)`?

Reply:
Mark as resolve. As long as it's working properly, we can plan to do a cleanup later